pkg/workflow_template.go

```diff
@@ -575,6 +656,35 @@ func (c *Client) ListWorkflowTemplateVersions(namespace, uid string) (workflowTe
 	return
 }
 
+// ListWorkflowTemplateVersionsModels returns all the WorkflowTemplateVersions from the database.
+// This function is a work-around for ListWorkflowTemplateVersions. Once that function is refactored,
+// this function should no longer be necessary and removed.
+// See: https://github.com/onepanelio/core/issues/436
+func (c *Client) ListWorkflowTemplateVersionsModels(namespace, uid string) (workflowTemplateVersions []*WorkflowTemplateVersion, err error) {
+	workflowTemplateVersions, err = c.listWorkflowTemplateVersionsDB(namespace, uid)
+	if err != nil {
+		log.WithFields(log.Fields{
+			"Namespace": namespace,
+			"UID":       uid,
+			"Error":     err.Error(),
+		}).Error("Workflow template versions not found.")
+		return nil, util.NewUserError(codes.NotFound, "Workflow template versions not found.")
+	}
+	return
+}
+
+// ListWorkflowTemplateVersionsDBAll returns all WorkflowTemplateVersions with no filtering.
```

Comment:
@aleksandrmelnikov typo in the method doc

`ListWorkflowTemplateVersionsDBAll` -> `ListWorkflowTemplateVersionsAll`

Reply:
Fixed